src/recommendations/RecommendationsPage.jsx

```diff
@@ -0,0 +1,132 @@
+import PropTypes from 'prop-types';
+
+import { DEFAULT_REDIRECT_URL } from '../data/constants';
+import messages from './messages';
+import RecommendationsList from './RecommendationsList';
+
+const recommendationData = [
+  {
+    key: 'course-key1',
+    title: 'Introduction to Blockchain',
+    bannerImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/894bd8b1-7f01-4004-946e-b29f3a3bb282-b656f554c388.small.png',
+    owners: [
+      {
+        key: 'test-owner-key1',
+        name: 'IBM',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/87b07564-d569-4cfd-bee6-8b0a407acb73-dc33e4b5f353.png',
+      },
+      {
+        key: 'test-owner-key12',
+        name: 'IBM1',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/87b07564-d569-4cfd-bee6-8b0a407acb73-dc33e4b5f353.png',
+      },
+    ],
+    marketingUrl: 'https://www.edx.org/course/introduction-to-blockchain',
+  },
+  {
+    key: 'course-key2',
+    title: 'Introduction to Blockchain',
+    bannerImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/894bd8b1-7f01-4004-946e-b29f3a3bb282-b656f554c388.small.png',
+    owners: [
+      {
+        key: 'test-owner-key2',
+        name: 'IBM',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/87b07564-d569-4cfd-bee6-8b0a407acb73-dc33e4b5f353.png',
+      },
+    ],
+    marketingUrl: 'https://www.edx.org/course/introduction-to-blockchain',
+  },
+  {
+    key: 'course-key3',
+    title: 'Introduction to Blockchain',
+    bannerImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/894bd8b1-7f01-4004-946e-b29f3a3bb282-b656f554c388.small.png',
+    owners: [
+      {
+        key: 'test-owner-key3',
+        name: 'IBM',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/87b07564-d569-4cfd-bee6-8b0a407acb73-dc33e4b5f353.png',
+      },
+    ],
+    marketingUrl: 'https://www.edx.org/course/introduction-to-blockchain',
+  },
+  {
+    key: 'course-key4',
+    title: 'Introduction to Blockchain',
+    bannerImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/894bd8b1-7f01-4004-946e-b29f3a3bb282-b656f554c388.small.png',
+    owners: [
+      {
+        key: 'test-owner-key4',
+        name: 'IBM',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/87b07564-d569-4cfd-bee6-8b0a407acb73-dc33e4b5f353.png',
+      },
+    ],
+    marketingUrl: 'https://www.edx.org/course/introduction-to-blockchain',
+  },
+];
+
+const RecommendationsPage = (props) => {
+  const { intl } = props;
+  const DASHBOARD_URL = getConfig().LMS_BASE_URL.concat(DEFAULT_REDIRECT_URL);
+
+  const handleSkip = (e) => {
+    e.preventDefault();
+    if (props.location.state && props.location.state.registrationResult) {
+      window.location.href = props.location.state.registrationResult.redirectUrl;
+    } else {
+      window.location.href = DASHBOARD_URL;
+    }
+  };
+
+  return (
+    <>
```

Comment:
No need for `<>`. It only has a single child that can be made the parent element.